spec/features/file_search_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'file viewer search bar', js: true do
+  include PurlFixtures
+  it 'limits shown files when text is entered' do
```

Comment:
Please use context blocks for each of these e.g. `context 'when text is entered' ...`